java/src/jmri/jmrix/can/cbus/CbusAddress.java

```diff
@@ -217,12 +217,12 @@ boolean match(CanMessage r) {
     /**
      * Split a string containing one or more addresses into individual ones.
      *
-     * @return null if entire string can't be parsed.
+     * @return 0 length if entire string can't be parsed.
```

Comment:
Thanks for changing all this to zero-length arrays. It might also be good to add @Nonnull annotations so that Spotbugs knows to enforce that. Thanks again.